modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectEntryServiceImpl.java

```diff
@@ -50,6 +50,19 @@
 )
 public class ObjectEntryServiceImpl extends ObjectEntryServiceBaseImpl {
 
+	@Override
+	public ObjectEntry addObjectEntry(
+			long userId, long groupId, long objectDefinitionId,
```

Comment:
is userId finally need it?

Reply:
The thing is that the manager exposes the userId everywhere and ObjectDDMStorageAdapter passes it from the Form. I think the current use of the adapter does not allow other uses but it's a general adapter, so if someone calls to adapt existing forms we should use the userId of the creator of the form, not the current user executing the adaptation.

So, to be safe/flexible I'm exposing it.